common/decrees/00_decree.txt

```diff
@@ -755,6 +753,7 @@ decree_military_training = {
 	modifier = {
 		state_conscription_rate_add = 0.05
 		building_training_rate_mult = 0.5
+		building_training_rate_add = 5
```

Comment:
A 50% increase wasn't enough already? Also I don't think this scales with building level? So it is just a flat 5 even when you have like a 50 level barracks.

Reply:
Do you know of a way to scale it? 50% of the lower tier training methods doesn't do much, which is why adding a base value to accelerate the training was important especially for lower tier armies.

Reply:
Only through production methods?